src/table/index.tsx

```diff
@@ -23,6 +23,7 @@ const Table = React.forwardRef(
         variant,
         wrapLines: props.wrapLines,
         enableKeyboardNavigation: props.enableKeyboardNavigation,
+        expandableRows: !!props.expandableRows,
```

Comment:
Is there a reason why we cast this one to a boolean while we hand the other data points as we get them?

Reply:
The expandable rows feature is active when the expandableRows property is defined. All four properties of expandable rows configuration are required and do not contribute to metrics, but we can extend it in the future if needed, for example:
```
expandableRows: !!props.expandableRows,
expandableRowsProgressiveLoading: !!props.expandableRows?.getItemState
```

Reply:
Very good point in this discussion. We can provide extra metadata, but not in this object. This object is only for vanilla props and I reverted this change: https://github.com/cloudscape-design/components/pull/2102

For custom metadata, you need to update `component-toolkit` to support `meta` field along with the `props`.

@pan-kot will you do that?